modules/database/audit-autoloaded-options/can-load.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Can load function to determine if Site Health module is supported or not.
+ *
+ * @since   n.e.x.t
+ * @package performance-lab
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+return static function () {
+	if ( ! has_filter( 'user_has_cap', 'wp_maybe_grant_site_health_caps' ) ) {
+		return new WP_Error( 'module_not_loaded', esc_html__( 'The module cannot be loaded with Performance Lab since it is disabled.', 'performance-lab' ) );
```

Comment:
Worth clarifying that it's _Site Health_ that's disabled. For example:
```suggestion
		return new WP_Error( 'cannot_load_module', esc_html__( 'The module cannot be loaded since the Site Health feature is disabled.', 'performance-lab' ) );
```